packages/pages-shared/__tests__/asset-server/handler.test.ts

```diff
@@ -1069,6 +1069,168 @@ describe("asset-server handler", () => {
 			);
 		});
 	});
+
+	test("should emit header when Web Analytics Token is injected", async () => {
+		const findAssetEntryForPath = async (path: string) => {
+			if (path === "/index.html") {
+				return "asset-key-index.html";
+			}
+			return null;
+		};
```

Comment:
NIT if we are using arrows then this can be more succinct:

```suggestion
		const findAssetEntryForPath = async (path: string) => path === "/index.html" ? "asset-key-index.html" : null;
```